packages/nav/Nav-item.tsx

```diff
@@ -0,0 +1,70 @@
+import styles from './Nav.scss'
+import { h, Component, prop } from 'skatejs';
+import { css } from '../../utils/css';
+
+const NavItemTypes = {
+  content: 'content',
+  success: 'success',
+  brand: 'brand',
+  info: 'info',
+  error: 'error',
+};
+type NavItemTypesType = typeof NavItemTypes;
+
+interface NavItemProps extends JSX.HTMLProps<HTMLElement | any> {
+  type?: keyof NavItemTypesType,
+  active?: boolean,
+  right?: boolean,
+  inline?: boolean,
+}
+export class NavItem extends Component<NavItemProps> {
+  _props: NavItemProps;
+
+  static get is() { return 'bl-nav-item' }
+  static get props() {
+    return {
+      type: prop.string({
+        attribute: true
+      }),
+      active: prop.boolean({
+        attribute: true
+      }),
+      right: prop.boolean({
+        attribute: true
+      }),
+      inline: prop.boolean({
+        attribute: true
+      }),
+    }
+  }
+
+  type: string;
+  active: boolean;
+  right: boolean;
+  inline: boolean;
+
+  renderCallback() {
+    const { type, active, right, inline } = this;
+    const className = css(
+      {
+        'c-nav__item--active': active,
+        'c-nav__item--right': right,
+        'c-nav__content': type === NavItemTypes.content,
```

Comment:
It seams a bit weird to have **content** type for item...

It can be used for images as noted here: http://blazecss.com/components/navs/
But I would consider having it as another element - **NavContent** or something...